Q/Qemu/build_tarballs.jl

```diff
@@ -152,15 +129,11 @@ products = [
 
 # Dependencies that must be installed before this package can be built
 dependencies = [
-    Dependency("Pixman_jll"),
+    Dependency("Pixman_jll"; compat="0.42.2, 0.43.4"),
```

Comment:
This is not true: https://github.com/JuliaPackaging/Yggdrasil/pull/8677#issue-229756727.  Our builds of 0.42.2 and 0.43.4 aren't ABI compatible on macOS < 12 because the build system changed, and that changed the compatibility version set in the Mach-O file.